modules/currency/currency.resolver.js

```diff
@@ -1,14 +1,47 @@
 const currencyService = require('./currency.service');
+const {
+  CURRENCY_NOT_FOUND,
+} = require('../../error-messages/currency.messages');
 
 const currencyQuery = {
   getAllCurrencies: () => currencyService.getAllCurrencies(),
-  getCurrencyById: (parent, args) => currencyService.getCurrencyById(args.id),
+  getCurrencyById: async (parent, args) => {
+    const currency = await currencyService.getCurrencyById(args.id);
+    if (currency) {
+      return currency;
+    }
+    return {
+      statusCode: 404,
+      message: CURRENCY_NOT_FOUND,
+    };
+  },
 };
 
 const currencyMutation = {
   addCurrency: (parent, args) => currencyService.addCurrency(args.currency),
-  deleteCurrency: (parent, args) => currencyService.deleteCurrency(args.id),
-  updateCurrency: (parent, args) => currencyService.updateCurrency(args.id, args.currency),
+  deleteCurrency: async (parent, args) => {
+    const currency = await currencyService.deleteCurrency(args.id);
+    if (currency) {
+      return currency;
+    }
+    return {
```

Comment:
Please, create a class that will take statusCode and message as a parameter to ensure that the same error message is used everywhere